### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by Marcraphael12**
In this PR, we are Connecting our app with the BookStore API

### CODE REVIEW

File: src/redux/books/thunk.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  dispatch(addBookAction(book));
+};
+
+export const removeBook = (id) => async (dispatch) => {
+  const res = await fetch(`${API}${id}`, {
+    method: 'DELETE',
+  });
+
+  dispatch(removeBookAction(id));
+};
```
Comment (ha-manel) [inline src/redux/books/thunk.js]: - [x] Kindly use `createAsyncThunk` for creating thunk action creators as mentioned in the project requirements, I see you imported it but it's not used anywhere 😅

File: src/redux/books/thunk.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
```
Comment (ha-manel) [inline src/redux/books/thunk.js]: - [x] Kindly avoid disabling linters because they're here to impose best practices and writing clean code 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Marcraphael12 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Professional README file ✔️

## Required Changes:

- [x] Kindly add the MIT license to your project because the link in your README is broken 👍 

_Kindly check the comments below_

## Optional:

- [x] Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters" or "linters are added" ... etc 👍 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/thunk.js (hunk 1 above)
Comment (Marcraphael12) [reply src/redux/books/thunk.js]: 😓let's go

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Marcraphael12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**